samples/nrf5340/netboot/README.rst

```diff
@@ -0,0 +1,82 @@
+.. _nc_bootloader:
+
+nRF5340 Network core bootloader
+################################
+
+This bootloader sample implements an immutable first stage bootloader that has the capability to update the application firmware on the network core of the nRF5340 SoC.
+In addition to this, the network core bootloader sample performs flash protection of both itself and the application.
+
+Overview
+********
+
+The network core bootloader sample supports transporting an already verified and authenticated firmware upgrade from the application core flash to the network core flash, as well as performing flash protection.
+
+This is accomplished by the following steps:
+
+1. Lock the flash of the bootloader.
+     The bootloader sample locks the flash that contains the sample bootloader and its configuration.
+     Locking is done using the ACL peripheral.
+     For details on locking, see the :ref:`fprotect_readme` driver.
+
+#. Check if there is a pending firmware upgrade of the application.
```

Comment:
I think this should say something more like "Perform any pending application firmware upgrade", as the upgrade itself is the more interesting feature (compared to checking for an upgrade)